.buildkite/hooks/post-command

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -euo pipefail
+[[ -n "${DEBUG:-}" ]] && set -x
+
+powershell -NoProfile -NonInteractive ci/purge.ps1 -projectRoot "$(pwd)/"
```

Comment:
Perhaps `&$PSScriptRoot"\purge.ps1" -projectRoot "$(pwd)/"`

Reply:
```line 5: syntax error near unexpected token `&'```

Reply:
and without the `&` I get:

```The system cannot find the path specified.```

Reply:
`powershell -NoProfile -NonInteractive ci/purge.ps1 -projectRoot "$(pwd)/"`

works on my local machine, I'm thinking this may be an agent environment issue